process/rating/ratingsData.go

```diff
@@ -0,0 +1,76 @@
+package rating
+
+import (
+	"github.com/ElrondNetwork/elrond-go/config"
+	"github.com/ElrondNetwork/elrond-go/process"
+)
+
+// RatingsData will store information about ratingsComputation
+type RatingsData struct {
+	startRating          uint32
+	maxRating            uint32
+	minRating            uint32
+	metaRatingsStepData  process.RatingsStepHandler
+	shardRatingsStepData process.RatingsStepHandler
+	selectionChances     []process.SelectionChance
+}
+
+// NewRatingsData creates a new RatingsData instance
+func NewRatingsData(settings *config.RatingsConfig) (*RatingsData, error) {
+	if settings.General.MinRating < 1 {
+		return nil, process.ErrMinRatingSmallerThanOne
+	}
+	if settings.General.MinRating > settings.General.MaxRating {
+		return nil, process.ErrMaxRatingIsSmallerThanMinRating
```

Comment:
might wrap this error in order to read the minimum/provided values

Reply:
fixed